app/_hub/kong-inc/mocking/_index.md

```diff
@@ -94,76 +95,124 @@ params:
       description: |
         Randomly selects one example and returns it. This parameter requires the spec to have multiple examples configured.
       minimum_version: "2.7.x"
+    - name: included_status_codes
+      required: false
+      default: null
+      datatype: array of integers
+      description: |
+        A global list of the HTTP status code that only allows being chosen and returned.
+      minimum_version: "3.1.x"
+    - name: random_status_code
+      required: false
+      default: false
+      datatype: boolean
+      description: |
+        Determines whether to randomly select an HTTP status code from the responses of the corresponding API method.
+        The default value is `false`, which means the minimum HTTP status code is always selected and returned.
+      minimum_version: "3.1.x"
   extra: |
 
     Depending on the Kong Gateway deployment mode, set either the `api_specification_filename`
     or the `api_specification` parameter. The plugin requires a spec to work.
 ---
 
-## Configuration
+## Behavioral Headers
 
-### Enable the plugin on a service
+Behavioral headers allow you to change the behavior of the Mocking plugin for the individual request without changing the configuration.
 
-Configure this plugin on a [service](/gateway/latest/admin-api/#service-object):
+### X-Kong-Mocking-Delay
+
+The `X-Kong-Mocking-Delay` header tells the plugin how many milliseconds to delay before responding. The delay value must be between `0`(inclusive) and `10000`(inclusive), otherwise a it returns a `400` error.
```

Comment:
```suggestion
The `X-Kong-Mocking-Delay` header tells the plugin how many milliseconds to delay before responding. The delay value must be between `0`(inclusive) and `10000`(inclusive), otherwise it returns a `400` error.
```